### DISCUSSION TITLE
How to pass by services like verifyemail.

### DISCUSSION THREAD

**Discussion by JasonMoriarty** [simple-login/app#1217] [Q&A] [answered]
https://verifymail.io/disposable-email

This service will verify the email address if it's a disposable email.

Some websites like JetBrains will reject email from simplelogin even it's a custom domain.

**Comment by nguyenkims** [comment, marked as answer]
@JasonMoriarty unfortunately some websites don't care to protect user info and we hope to be able to change that. I tried to submit a request on https://verifymail.io/signup but they don't allow an alias neither and I would never trust a website like this with my real email unfortunately.

**Comment by zeroav-org** [comment]
~~Best solution is to buy a domain from a cheap TLD and set it up as a custom domain in simplelogin.~~

**Reply by JasonMoriarty** [reply]
Yes but no. it doesn't just check the domain. But MX record. So if you put your custom domain in simplelogin they still gonna get you.

**Reply by ghost** [reply]
There are a couple of ways you could view this.

1) Vote with your wallet and don’t sign up to any website or service that does not respect your privacy, why would you want to?

2) I have just setup a new verifyemail account, using a randomly generated Mozilla relay mask address, and that works just fine.

I know from trying to keep up with what is going on in the privacy arena, Mozilla actively keep up with getting their domains off block and ban lists.

Its not an ideal solution, personally I’d rather use just use SimpleLogin. But, in my opinion, any website or service that won’t accept my personal domain with a SimpleLogin MX record can jog on as far as I’m concerned. I'm not interested in doing business with them, no matter how badly I want or need their service.

There has to be a line, a line you define and are ultimately responsible for, its your data after all. The more people accept this type of behaviour as ‘normal’ and ‘accepted’ the more normal and accepted it will become.
